config/coq_config.mli

```diff
@@ -80,4 +80,5 @@ val wwwbugtracker : string
 val wwwstdlib : string
 val localwwwrefman : string
 
+val enable_vm : bool
```

Comment:
Are you sure you want this to be a static flag? I remember that there was a similar VM option at some point.